node/src/components/chain_synchronizer/operations.rs

```diff
@@ -1081,31 +1072,5 @@ mod tests {
             }
             result => panic!("expected future block version error, got {:?}", result),
         }
-
-        // If the current version is 1.4.0 but the current block's is 1.3.0, we have to downgrade.
```

Comment:
If we're removing this unit test, shouldn't we also be removing some logic that was causing the downgrade?

Reply:
We are! That code is removed from `check_block_version` above.

Reply:
🙈 somehow I totally mised that